MediaGalleryUi/view/adminhtml/templates/image_edit_details_standalone.phtml

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+use Magento\Backend\Block\Template;
+
+// phpcs:disable Magento2.Files.LineLength, Generic.Files.LineLength
+/** @var Template $block */
+/** @var \Magento\Framework\Escaper $escaper */
+?>
+
+<div class="media-gallery-edit-image-details-modal"
+     data-bind="mageInit: {
+        'Magento_Ui/js/modal/modal': {
+            type: 'slide',
+            buttons: [],
+            modalClass: 'media-gallery-edit-image-details',
+            title: '<?= $escaper->escapeHtmlAttr(__('Edit Image')); ?>'
+        }
+     }">
+    <div class="page-main-actions">
+        <div class="page-actions">
+            <div class="page-actions-inner">
+                <div class="page-action-buttons" id="media-gallery-edit-image-actions"
+                     data-bind="scope: 'mediaGalleryImageEditActions'">
+                    <!-- ko template: getTemplate() --><!-- /ko -->
+                </div>
+            </div>
+        </div>
+    </div>
+    <div id="media-gallery-image-edit-details" data-bind="scope: 'mediaGalleryEditDetails'">
+        <!-- ko template: getTemplate() --><!-- /ko -->
+    </div>
+</div>
+
+<script type="text/x-magento-init">
+    {
+        "#media-gallery-image-edit-details": {
+            "Magento_Ui/js/core/app": {
+                "components": {
+                    "mediaGalleryEditDetails": {
+                        "component": "Magento_MediaGalleryUi/js/image/image-edit",
+                        "imageEditDetailsUrl": "<?= $escaper->escapeJs($block->getData('imageEditDetailsUrl')); ?>",
+                        "saveDetailsUrl": "<?= $escaper->escapeJs($block->getData('saveDetailsUrl')); ?>",
+                        "modalSelector": ".media-gallery-edit-image-details-modal",
+                        "mediaGridMessages": "standalone_media_gallery_listing.standalone_media_gallery_listing.messages"
+                    }
+                }
+            }
+        },
+        "#media-gallery-image-details-messages": {
+            "Magento_Ui/js/core/app": {
+                "components": {
+                    "mediaGalleryImageDetailsMessages": {
```

Comment:
I think this component should have a unique name to avoid potential conflicts with the image details component
```suggestion
                    "mediaGalleryEditDetailsMessages": {
```